### PR TITLE
ocrd_eval: add alternative order metrics

### PR DESCRIPTION
**Pull request by bertsky**

### CODE REVIEW

File: site/en/spec/ocrd_eval.md [hunk 1]
```diff
@@ -382,6 +382,13 @@ The success measure is then given by
 
 $s = \frac{1}{e * (1/e_{50}) + 1}$
 
+Furthermore, in the special case where…
```
Comment (mweidling) [inline site/en/spec/ocrd_eval.md:385]:
```suggestion
Furthermore, in the special cases where…
```
?

File: site/en/spec/ocrd_eval.md [hunk 2]
```diff
@@ -382,6 +382,13 @@ The success measure is then given by
...
+- the segmentation on both sides is identical (only order differs → no matching needed)
+- order is always strictly monotonic (no unordered subgroups involved → no misclassification other than successor vs predecessor)
+
+…[Quirós and Vidal 2022](https://ieeexplore.ieee.org/document/9413256) evaluate reading order simply as rank distance,
+using the metrics _Normalized Spearman's footrule distance_ and _Kendall's Tau rank distance_.
```
[the remark this answers is not in the document]
Comment (bertsky) [reply site/en/spec/ocrd_eval.md:390]: No, they are not so common AFAICT, but the general principle (_rank distance_) ist, and as long as we are not using these metrics ourselves, I thought it suffices to just reference the paper.

[the remark this answers is not in the document]
Comment (mweidling) [reply site/en/spec/ocrd_eval.md:390]: Agreed, I just wanted to clarify this.

**Comment by mweidling** [comment]
AFAICT this PR should rather point at https://github.com/OCR-D/ocrd-website/tree/master since the files in `spec` get imported while building the website. `site` contains just a copy of them.

Would you open a PR over there or should I tend to that?

**Comment by bertsky** [comment]
> AFAICT this PR should rather point at https://github.com/OCR-D/ocrd-website/tree/master since the files in `spec` get imported while building the website. `site` contains just a copy of them.

You mean https://github.com/OCR-D/spec?

**Comment by mweidling** [comment]
> > AFAICT this PR should rather point at https://github.com/OCR-D/ocrd-website/tree/master since the files in `spec` get imported while building the website. `site` contains just a copy of them.
> 
> You mean https://github.com/OCR-D/spec?

Yep :D

**Comment by mweidling** [comment]
Closed in favour of https://github.com/OCR-D/spec/pull/247.

**Pull request closed without merging**